Assets/Python/CityNameManager.py

```diff
@@ -5626,14 +5706,84 @@ def onRevolution(iCiv):
 #Language: Korean
 {
 	"Beijing"		:	"Bukgyeong",
-	"Zhongdu"		:	"Bukgyeong",
+	"Zhongdu"		:	"Bukpyeong",

```

Comment:
This looks like a typo.

Reply:
Busan? No, that's the official English translation of the city's name.